test.js

```diff
@@ -346,6 +346,29 @@ describe('Gfycat JS SDK', function () {
         });
       });
     });
+
```

Comment:
Is it the same test but in two different places?

Reply:
Yeah, except one of them uses promises